lib/radiator/directory/editor.ex

```diff
@@ -458,4 +458,89 @@ defmodule Radiator.Directory.Editor do
       @not_authorized_match
     end
   end
+
+  ## User Permission Management
+
+  # TODO: list all collaborators of underlying entities as well.
+
+  @spec list_collaborators(Auth.User.t(), Network.t()) ::
```

Comment:
If there's an `{:error, _}` reply, the happy path is usually `{:ok, _}`. Not 100% sure about the convention but I can't think of an example where that's not the case.

Reply:
ah i see, was queuing from `Editor.list_podcasts` - although that doesn't handle the authentication yet either. So I take it we'll put everhting editor as {:ok, }. Need to audit a little probably.